src/main/java/domain/Car.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+import java.util.Random;
+
+public class Car {
+    public static final int INITIAL_POSITION = 1; //위치 기본값 상수
+    public static final int MOVE_VALUE = 4;//움직임 기준 상수
+    private final String name;
+    private int position;
+
+    public Car(String name) { //Car 객체 생성
+        this.name = name;
+        this.position = Car.INITIAL_POSITION;
+    }
+
+    public String getNames() {
```

Comment:
사소하지만, 복수형으로 제공하면 `Collection`을 반환하는 것처럼 보일 수 있습니다.
```suggestion
    public String getName() {
```